icaro.js

```diff
@@ -110,6 +110,9 @@ const API = {
    * @returns {API}
    */
   listen(fn) {
+    if(typeof fn !== "function"){
```

Comment:
please use `if (typeof fn !== fn ) return this` to keep the api consistent and avoid the curly braces ;)